src/librustdoc/clean/types.rs

```diff
@@ -1565,23 +1566,11 @@ impl Type {
 
     /// Use this method to get the [DefId] of a [clean] AST node, including [PrimitiveType]s.
     ///
-    /// See [`Self::def_id_no_primitives`] for more.
-    ///
     /// [clean]: crate::clean
+    ///
```

Comment:
Why this add?

Reply:
Most of this PR looks good, but I’m also not sure about this added blank line.